packages/components/pie-button/src/button.scss

```diff
@@ -6,46 +6,6 @@
     box-sizing: border-box;
 }
 
-/**
```

Comment:
moved this to `pie-css` so we can reuse it in the icon-button component.

Reply:
are we certain they are exactly the same between components?

Reply:
yes. 💯